environment.yml

```diff
@@ -1,38 +1,29 @@
-name: xcube
+name: xcube5
 channels:
   - conda-forge
-  - defaults
 dependencies:
   # Python
-  - python >=3.6
-  # Required
+  - python
   - affine >=2.2
-  - blas =*=mkl
   - click >=7.0
   - cmocean >=2.0
   - dask >=1.2
   - fiona >=1.8
-  - gdal >=2.4,<3.0
-  - matplotlib >=3.0
+  # - gdal >=2.4
+  - matplotlib >=3.0.3
   - netcdf4 >=1.5.0
   - numba >=0.43
-  - numpy >=1.16
+  # - numpy >=1.16
   - pandas >=0.24
   - pillow >=6.0
-  - proj4 >=5.2
-  - pyproj >=2.1
+  # - proj4 >=5.2
```

Comment:
Please comment, what you did here. Why are `proj4 ` and `pyproj` no longer in?

Reply:
Only change now is xarray version